### PR TITLE
Java8 plus deprecation

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by Bo98** [comment]
It's worth checking if `fits` is actually using brewed OpenJDK. Might need to insert `JAVA_HOME` somewhere.

File: Formula/giter8.rb [hunk 1]
```diff
@@ -7,7 +7,10 @@ class Giter8 < Formula
 
   bottle :unneeded
 
-  depends_on java: "1.8+"
+  # Doesn't actually build from source
```
Comment (bayandin) [inline Formula/giter8.rb:10]: What do you mean by "build from source"?

It's `bottle :unneeded` formula, and we don't build such kind of formulae usually

Comment (SMillerDev) [reply Formula/giter8.rb:10]: Yeah, that's bad naming. It downloads a shell script that downloads the actual software and dependencies. The change in https://github.com/Homebrew/homebrew-core/pull/50160 shows what the script does

**Comment by SMillerDev** [comment]
> It's worth checking if `fits` is actually using brewed OpenJDK - I'm sceptical of whatever they are doing in their shell scripts. Might need to insert `JAVA_HOME` somewhere.

I don't have the cask installed and it passed tests locally

File: Formula/civl.rb [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
-    libexec.install "lib/civl-1.7_3157.jar"
-    bin.write_jar_script libexec/"civl-1.7_3157.jar", "civl"
+    underscored_version = version.to_s.tr("-", "_")
+    libexec.install "lib/civl-#{underscored_version}.jar"
+    bin.write_jar_script libexec/"civl-#{underscored_version}.jar", "civl"
     pkgshare.install "doc", "emacs", "examples", "licenses"
   end
 
   test do
+    ENV["HOME"] = testpath
```
Comment (bayandin) [inline Formula/civl.rb:22]:
```suggestion
```

We don't need to do it explicitly, brew sets it automatically during the test run:
https://github.com/Homebrew/brew/blob/197c4666cf1b53fa3e59d97347fefac73bf88a98/Library/Homebrew/formula.rb#L1825

**Review by bayandin — CHANGES_REQUESTED** [review]

**Review by bayandin — APPROVED** [review]

**Pull request merged**